WebPhapp/backend/server.js

```diff
@@ -362,7 +362,7 @@ Returns:
 Warning:
     no validation exists for blockchain index yet. See Issue #32 on GitHub.
 */
-app.get('/api/v1/prescriptions/single/:prescriptionID', (req,res) => {
+app.get('/api/v1/prescriptions/single/:prescriptionID', auth.checkAuth([Role.Patient, Role.Prescriber, Role.Government, Role.Dispenser]), (req,res) => {
```

Comment:
Will need validation in the patient role.

Reply:
I don't know how to test your JWT stuff yet, so I don't know how to extract PatientID from the request (if it is even present yet).